backend/src/main/java/de/ffmjava/capstone/backend/horses/Horse.java

```diff
@@ -0,0 +1,16 @@
+package de.ffmjava.capstone.backend.horses;
+
+import lombok.With;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+
+@With
+public record Horse(
+        @Id
+        String id,
+        String name,
+        String owner,
+        List<Consumption> dailyConsumption
```

Comment:
Maybe use some validation, so there is no Horse without name e.g.

Reply:
Will implement later with the ADD endpoint